src/components/Card/Card.story.jsx

```diff
@@ -46,6 +46,9 @@ storiesOf('Watson IoT/Card', module)
           breakpoint="lg"
           availableActions={{ range: true, expand: true }}
           onCardAction={action('onCardAction')}
+          onFocus={() => console.log('onFocus')}
```

Comment:
we should log to the action panel here I think

Reply:
+1

```suggestion
          onFocus={action('onFocus')}
```